tests/test_ingest.py

```diff
@@ -259,247 +259,247 @@ def test_ingest_invalid_compact_payload(self):
-                tile_tested += 1
-
-            if ag.end_date:
-                assert_equal(ag.end_date, tile_def['time_limits']['end'])
-                tile_tested += 1
-
-        # one tile not tested because it has neither start or end dates
-        assert_equal(len(dist["US/en-US"]) - 1, tile_tested)
-
-    def test_start_end_dates_uniqueness(self):
-        """
-        Test that start/end are part of what make tiles unique
-        """
-        tile = {
-            "imageURI": "data:image/png;base64,somedata",
-            "url": "https://somewhere.com",
-            "title": "Some Title",
-            "type": "organic",
-            "bgColor": "#FFFFFF",
-            "time_limits": {
-                "start": "2014-01-12T00:00:00.000",
-                "end": "2014-01-31T00:00:00.000"
-            }
-        }
-
-        tile_no_start = {
-            "imageURI": "data:image/png;base64,somedata",
-            "url": "https://somewhere.com",
-            "title": "Some Title",
-            "type": "organic",
-            "bgColor": "#FFFFFF",
-            "time_limits": {
-                "end": "2014-01-31T00:00:00.000"
-            }
-        }
-
-        tile_no_end = {
-            "imageURI": "data:image/png;base64,somedata",
-            "url": "https://somewhere.com",
-            "title": "Some Title",
-            "type": "organic",
-            "bgColor": "#FFFFFF",
-            "time_limits": {
-                "start": "2014-01-12T00:00:00.000",
-            }
-        }
-
-        tile_empty_limits = {
-            "imageURI": "data:image/png;base64,somedata",
-            "url": "https://somewhere.com",
-            "title": "Some Tile",
-            "type": "organic",
-            "bgColor": "#FFFFFF",
-            "time_limits": {}
-        }
-
-        dist = {"US/en-US": [tile, tile_no_start, tile_no_end, tile_empty_limits]}
-        c = self.env.db.session.query(Adgroup).count()
-        assert_equal(30, c)
-        data = ingest_links(dist, self.channels[0].id)
-        assert_equal(len(dist["US/en-US"]), len(data["US/en-US"]))
-        c = self.env.db.session.query(Adgroup).count()
-        assert_equal(30 + len(dist["US/en-US"]), c)
-
-    def test_adgroups_channel_id_uniqueness(self):
-        """
-        Test that channel_ids in adgroups are part of what makes Tiles unique
-        """
-        tile = {
-            "imageURI": "data:image/png;base64,somedata",
-            "url": "https://somewhere.com",
-            "title": "Some Title",
-            "type": "organic",
-            "bgColor": "#FFFFFF",
-        }
-
-        dist = {"US/en-US": [tile]}
-        c = self.env.db.session.query(Adgroup).count()
-        assert_equal(30, c)
-        ingest_links(dist, self.channels[0].id)
-        ingest_links(dist, self.channels[1].id)
-        c = self.env.db.session.query(Adgroup).count()
-        assert_equal(32, c)
+#    def test_start_end_dates(self):
```

Comment:
we can always come back to this branch if we need this code.  Let's just remove it.